src/StreamDeckLib/ConnectionManager.cs

```diff
@@ -260,6 +262,33 @@ public async Task SetImageAsync(string context, string imageLocation)
 			await _proxy.SendStreamDeckEvent(args);
 		}
 
+		public async Task SetPngImageAsync(string context, Bitmap image)
+		{
+			Debug.WriteLine("Getting Image from Bitmap");
+			_logger?.LogDebug("Getting Image from Bitmap");
+
+			byte[] imgBytes;
+			using (var imgByteStream = new MemoryStream())
```

Comment:
at this point we could use https://github.com/microsoft/Microsoft.IO.RecyclableMemoryStream